templates/layout/region--header.html.twig

```diff
@@ -0,0 +1,11 @@
+{% embed "@organisms/site-header/site-header.twig" with {
+  site_name: site_name,
+  site_header__border_thickness: '8',
+} %}
+  {% block site_header__utility_nav %}
+    {{ elements.user_account_menu }}
```

Comment:
I think this is already on your to do list. Let's create a new menu 'Utility Menu'. This will change the name of the drupal template from `menu--account` to `menu--utility`.

Reply:
Done.